src/axios/interceptor.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosError, AxiosResponse, InternalAxiosRequestConfig } from 'axios';
+const status: { [key: number]: string } = {
+  400: '데이터필드를 확인해주세용',
+  404: '페이지를 찾을 수 없습니다.',
+  500: '서버 오류가 발생했습니다.',
+};
```

Comment:
```suggestion
const status = {
  400: '데이터필드를 확인해주세용',
  404: '페이지를 찾을 수 없습니다.',
  500: '서버 오류가 발생했습니다.',
} as const;
```
as const로 관리는 힘든걸까요?